assets/sass/components/wp-dashboard/googlesitekit-pointers.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * Pointers styles for WP Dashboard
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-email-pointer {
+	.wp-pointer-content h3::before {
+		content: "\f466"; // Email icon
+		font-family: dashicons, sans-serif;
+		-webkit-font-smoothing: antialiased;
+		-moz-osx-font-smoothing: grayscale;
```

Comment:
- Why are these smoothing values different?
- We shouldn't be authoring browser prefixed styles since PostCSS should handle this for us according to our Browserslist config